packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -120,6 +132,13 @@ class DebuggerController extends DisposableController
     commonScriptPrefix = scriptPrefix;
   }
 
+  /// Append to the stdout / stderr buffer.
+  void appendStdio(String text) {
+    // TODO(devoncarew): Check for greater than some amount of text and truncate
```

Comment:
why should we truncate?

Reply:
Just in case the amount of text gets really large? 10k lines? 100k? I'm not sure how much we want to retain, and when we'd start having issues, either with the UI or the VM we're running on.

Reply:
makes sense